AssetInformationApi/V1/Boundary/Request/AddAssetRequest.cs

```diff
@@ -0,0 +1,8 @@
+using Hackney.Shared.Asset.Domain;
+
+namespace AssetInformationApi.V1.Boundary.Request
+{
+    public class AddAssetRequest : Asset
```

Comment:
Why do we have an empty request object? 
The request object should be in the Asset Shared Nuget Package